### DISCUSSION TITLE
Simplify the configuration path directory name

### DISCUSSION THREAD

**Discussion by Ioulan** [johnfactotum/foliate#1214] [Ideas]
From `com.github.johnfactotum.Foliate` to just `foliate`.

(Just like renaming the binary into `foliate` for consistency.)

**Comment by johnfactotum** [comment]
It would be more of a breaking change compared to renaming the executable. So might not be worth it at this point.

If it does change, though, it would need to make WebKitGTK save cache to a different directory than the one with the app ID. Apparently it's possible but I'm not sure how.
